session.go

```diff
@@ -3063,6 +3063,26 @@ func (c *Collection) UpdateId(id interface{}, update interface{}) error {
 	return c.Update(bson.D{{Name: "_id", Value: id}}, update)
 }
 
+func (c *Collection) UpdateWithArrayFilters(selector, update, arrayFilters interface{}, multi bool) (*ChangeInfo, error) {
```

Comment:
Hi, could you please add a short comment here explaining the `arrayFilters` and `multi` parameters?
Something that would make clear how `UpdateWithArrayFilters` is different than normal `Update`?

Having even short comments on public methods and fields is very helpful when browsing godoc documentation.

Also, information that this function is not compatible with mongodb <3.6 would be greatly appreciated.

Reply:
Thanks for the comments. I will add what you mentioned tonight.